.github/workflows/test.yaml

```diff
@@ -71,7 +71,10 @@ jobs:
 
             - id: components
               run: |
-                components=$(tree src -J -d -L 1 | jq -c '.[0].contents | map(.name)')
+                components=()
+                while IFS= read -r line; do
+                components+=("$line")
+                done <<< $(find src -type f -name composer.json \( -path 'src/*/composer.json' -o -path 'src/*/src/Bridge/*/composer.json' \) ! -path '*/vendor/*' | sed -e 's/^src\///' -e 's/\/composer.json$//' | sort)
```

Comment:
Didn't find a way to do that with `tree`, but this modification add `Map/src/Bridge/Google` and `Map/src/Bridge/Leaflet` to the components array.

Reply:
But... looks like it does not work in the CI 🥲 
<img width="1151" alt="image" src="https://github.com/user-attachments/assets/990244b7-6bda-415f-bc39-c439775aa37f">

Reply:
It's working :) 
<img width="1007" alt="image" src="https://github.com/user-attachments/assets/def91b3c-1b07-4a36-9200-fb26a388a041">